eva-lib/src/main/java/uk/ac/ebi/eva/lib/metadata/shared/ArchiveWSServerHelper.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.lib.metadata.shared;
+
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.springframework.stereotype.Service;
+
+import uk.ac.ebi.eva.lib.metadata.ArchiveDBAdaptor;
+import uk.ac.ebi.eva.lib.metadata.StudyDBAdaptor;
+import uk.ac.ebi.eva.lib.utils.QueryOptions;
+import uk.ac.ebi.eva.lib.utils.QueryResponse;
+import uk.ac.ebi.eva.lib.utils.QueryResult;
+import uk.ac.ebi.eva.lib.utils.QueryUtils;
+
+import java.util.List;
+import java.util.Map;
+
+@Service
```

Comment:
In this case I don't see any reason to use it. It can incur a performance penalty and it is a completely static dependency, so there is no need for injection.